src/NzbDrone.Core/NetImport/NetImportSearchService.cs

```diff
@@ -186,10 +190,36 @@ private void CleanLibrary(List<Movie> movies)
                         if (movie.TmdbId == listedMovie.TmdbId)
                         {
                             foundMatch = true;
+                            switch (_configService.ListSyncLevel)
+                            {
+                                case "logOnly":
+                                    _logger.Info("{0} was in your library and found in your lists --> You might want to unmonitor or remove it", movie);
+                                    break;
+                                case "keepAndUnmonitor":
+                                    _logger.Info("{0} was in your library and found in your lists --> Keeping in library but Unmonitoring it", movie);
+                                    movie.Monitored = false;
+                                    break;
+                                case "removeAndKeep":
+                                    _logger.Info("{0} was in your library and found in your lists --> Removing from library (keeping files)", movie);
+                                    _movieService.DeleteMovie(movie.Id, false, true);
+                                    break;
+                                case "removeAndDelete":
+                                    _logger.Info("{0} was in your library and not found in your lists --> Removing from library and deleting files", movie);
+                                    _movieService.DeleteMovie(movie.Id, true, true);
```

Comment:
Don’t know if this one belongs here, if we are getting into this code it means the movie is in the library and on the list (foundMatch = true). So we shouldn’t be deleting it here right?

Reply:
I dont think the code was in the wrong position..... I agree that perhaps the removeAndKeep or removeAndDelete options can be dangerous but they should only be used if peopel know what they are doing.. I personally use the logOnly aspect of this feature.. After NetImport Sync radarr will log all movies that dont exist in my synced lists but do exist in my radarr library. I then manually delete the movies from my radarr movie....

My point is, I think the code was in the right place and correct prior to these changes.... Having said that, it might be nice to actually have a place in radarr to keep track of and manage these things..  Instead of the various options (automated or logging) somewhere in radarr there could be a table of entries (corresponding to what used to result from logOnly with the ability to then delete the movies from there)... that is wishful thinking. For now, I am happy withe logOnly and manual deleting, I just put the other options in there for completeness, or total automation if anybody was certain/confident enough in automating it to do it... 

it is why the herlper popout beside the option says "only enable if you know what you are doing - recycle bin is reccomended before enabling this"